openfl/interface/workspace.py

```diff
@@ -450,9 +450,12 @@ def dockerize_(context, base_image, save):
         default=True, type=bool,
         help='Dump the Docker image to an archive')
 @option('--rebuild', help='Build images with `--no-cache`', is_flag=True)
+@option('--developer-mode', required=False,
+        default=False, type=bool, is_flag=True,
+        help='Use the local OpenFL package in the Docker image')
 @pass_context
 def graminize_(context, signing_key: Path, enclave_size: str, pip_install_options: Tuple[str],
-               save: bool, rebuild: bool) -> None:
+               save: bool, rebuild: bool, developer_mode: bool) -> None:
```

Comment:
`COPY . .` in the Dockerfile will not work for this command. The reason is `fx workspace graminize` command, as well as `fx workspace dockerize` are designed to be called from an FL workspace, not OpenFL repo.
I imagine the additional parameter to be either a local path to OpenFL repo or a URL for a remote repo.
Something like `openfl_source: Union[Path, str]`, with additional parameter `branch` for the remote option

Reply:
Got it. We could do this with 1 parameter: the branch could be specified in the URL itself.